### PR TITLE
added delete document function

### PR DESCRIPTION
**Pull request by alizhd**
Deletes document with an ID. Tested it and it worked

### CODE REVIEW

File: hyacinth/session.py [hunk 1]
```diff
@@ -199,6 +199,11 @@ def download_document(self, id, **kwargs):
         """Download a Document with provided ID."""
         url = Session.__make_url(f"documents/{id}/download")
         return self.__get_resource(url, **kwargs)
+    
+    def delete_document(self, id, **kwargs):
+        """DELETE an existing DOCUMENT with provided ID."""
```
Comment (amackera) [inline hyacinth/session.py:204]:
```suggestion
        """DELETE an existing Document with provided ID."""
```

**Review by amackera — APPROVED** [review]
Looks great! Thanks for submitting this 🔥

**Review by amackera — APPROVED** [review]
Looks great! Thanks for submitting this 🔥

**Comment by alizhd** [comment]
@amackera can you take a quick look again?

**Review by amackera — APPROVED** [review]
L
G
T
M

:shipit:

**Pull request merged**